rootaccess2023/portfolio
# Work

This pull request introduces multiple updates and enhancements across the app, with a primary focus on improving component functionality, responsiveness, and UI consistency. Key changes include:

-  Responsive Design Improvements: Adjusted the layout and styles in WorkProjectList and the entire Work page to ensure optimal viewing across various screen sizes.
- New Components and Hooks:
- Created WorkDropdown and WorkProjectList components to organize and display projects effectively.
- Added a custom useProjectFilter hook to handle project filtering logic.
- Theme and State Enhancements:
- Integrated headerColor into the contact SVG icon for consistent theming.
- Added an isOpen condition in App sourced from menuStore, enhancing menu state management.
- Asset and Styling Updates:
- Imported and displayed Fare Finder poster and GIF within WorkProjectList.
- Updated font sizes, padding, and color styles in WorkProjectList for better readability and visual alignment.

How to Test

- Responsive Design: Check the Work page and WorkProjectList component across different screen sizes to confirm responsive adjustments.
- New Functionalities:
- Toggle the menu state in App to verify the isOpen condition works as expected.
- Use the project filter within WorkProjectList to test the useProjectFilter hook.
- UI Consistency: Review the headerColor changes in the contact SVG icon for alignment with the app’s color scheme.

Notes

This PR refines the app's structure, making components more modular and reusable. It also enhances the user experience on mobile and desktop views.
